src/controllers/v1/org-admin.js

```diff
@@ -125,7 +125,11 @@ module.exports = class OrgAdmin {
 	 */
 	async updateRelatedOrgs(req) {
 		try {
-			return await orgAdminService.updateRelatedOrgs(req.body.related_organization_ids, req.body.organization_id)
+			return await orgAdminService.updateRelatedOrgs(
+				req.body.delta_organization_ids,
```

Comment:
Confirm that this is a strictly internal api. Should always require a internal access token.